specs/APIReview_AllowExternalDrop.md

```diff
@@ -0,0 +1,100 @@
+# Background
+Currently the functionality of dragging and dropping external objects(e.g. files, hyperlinks) into webview2 is default enabled in webview2 and there is no way to disable it. Some developers may want to disbale this functionality in their applications based upon their scenarios. According to such feature requirements, we add the new API to provide developers with the capability to configure the external drag&drop functionality.
+
+# Description
+We add a new `AllowExternalDrop` property in `CoreWebView2Controller`. 
+This API allows end developers to toggle the external drag&drop functionality easily.
+If it's disabled, drag&drop objects from outside into current webview2 will be forbidden.
+To be more specific, some behaviors listed below will be impacted by the toggle of this property.
+
+* Drag&Drop files on disk to webview2  
+* Drag&Drop hyperlinks from browser to webview2
+* Drag&Drop hyperlinks from one webview2 to another webview2
+
+Some behaviors are not impacted by the toggle of this property like drag&drop selected text to webview2. That means toggle of this property has no impact on the expected behavior of this action.
+
+By default, AllowExternalDrop is enabled to keep consistent with the behavior we had before the API is added.
+
+Please note that drag&drop anything from webview2 to external(outside current webview2) will not be impacted by this property.
+
+# Examples
+## C++
+
+```cpp
+wil::com_ptr<ICoreWebView2Controller4> m_controller4;
+void ToggleAllowDrop()
+{
+    if (m_controller4)
+    {
+        BOOL allowDrop;
+        CHECK_FAILURE(m_controller4->get_AllowDrop(&allowDrop));
```

Comment:
AllowDrop needs to be updated to AllowExternalDrop throughout this sample.

Reply:
Thanks. All AllowDrop are replaced to AllowExternalDrop.